src/toil/jobStores/aws/jobStore.py

```diff
@@ -47,6 +49,47 @@
 
 log = logging.getLogger(__name__)
 
+s3partSize = 2**20*50 
```

Comment:
Lets call it defaultS3PartSize to indicate that it can be overriden. Also makes it S3 instead of s3.